mod_reforged/actor_tooltip_functions.nut

```diff
@@ -271,6 +271,24 @@
 	local ret = [];
 
 	local skills = _actor.getSkills().getAllSkillsOfType(::Const.SkillType.Active);
+	if (!_actor.isPlayerControlled())
```

Comment:
I suggest adding a comment for this block summarizing what is being done. Could be a commend in front of this if with an empty line above it.